### PR TITLE
Changes done in the all the appropriate model files according to the …

### PR DESCRIPTION
**Pull request by techspawn1**
…new version of api available in woocommerce version 2.6.4

Fully compatible with woocommerce version(2.4.6)
Modified the process of import to get the list of the data
Modified the date time format in WOO_DATETIME_FORMAT

### CODE REVIEW

**Comment by parthivgls** [comment]
Not possible to merge as at a first glance original author's copyright is blatantly removed and replaced.

File: connector_woocommerce/__init__.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
 #
 #
-#    Tech-Receptives Solutions Pvt. Ltd.
-#    Copyright (C) 2009-TODAY Tech-Receptives(<http://www.techreceptives.com>).
+#    TechSpawn Solutions Pvt. Ltd.
```
Comment (parthivgls) [inline connector_woocommerce/__init__.py:4]: what is the reason for removing copyright ?

File: connector_woocommerce/__openerp__.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
 #
 #
-#    Tech-Receptives Solutions Pvt. Ltd.
-#    Copyright (C) 2009-TODAY Tech-Receptives(<http://www.techreceptives.com>).
+#    TechSpawn Solutions Pvt. Ltd.
```
Comment (parthivgls) [inline connector_woocommerce/__openerp__.py:4]: what is the reason for removing copyright ?

File: connector_woocommerce/__openerp__.py [hunk 3]
```diff
@@ -16,15 +17,14 @@
...
-#
+
 
 {
     'name': 'WooCommerce Connector',
-    'version': '8.0.1.0.1',
+    'version': '8.0.1.0.2',
     'category': 'customized',
-    'description': """WooCommerce Connector.""",
-    'author': 'Tech Receptives',
```
Comment (parthivgls) [inline connector_woocommerce/__openerp__.py:27]: pure show of plagiarism

**Comment by parthivgls** [comment]
closing this PR as to me is just for own benefit purpose instead of working as per community standards.

**Comment by techspawn1** [comment]
@parthivgls As it was our first commit and it was not meant for own benefits. We just wanted to enhance and add extra compatibility for the newer version of woo commerce i.e 2.6.4 and contribute it to the community. Can you please advice best possible way to do it and also get credits without impacting original copy rights.

**Comment by pedrobaeza** [comment]
You should add copyright lines instead of replacing in the files (and not all), and only in the affected.

You should also take into account if the same backend is compatible for all versions or only the newer ones. If not, you need to create two backends.

**Pull request closed without merging**